src/main/resources/templates/privacyPolicy.html

```diff
@@ -0,0 +1,81 @@
+            in accordance with the statutory data protection regulations and this data protection declaration.<br>
+            You can use this website without providing any personal data. We collect personal data such as name, address or
+            email address on a voluntary basis. This information shall not be disclosed to third parties without your explicit
+            consent.<br>
+            Data transmission over the Internet can have security issues, e.g. via communication by email. It is not possible
+            to fully protect data from third-party access.
+        </p>
+        <h4>General information</h4>
+        <p>
+            When you access our website, information of a general nature is gathered automatically. This information
+            (server log files) includes, among others things, the type of browser used, the operating system used, the
+            domain name of your internet service provider, and other similar data. This is only about information that
+            cannot be connected back to you as an individual. This information is produced unavoidably during internet
+            use and is required for technical reasons in order to display the requested web content correctly. Anonymous
+            information of this kind can be statistically analysed in order to improve our website and its underlying technology.
+        </p>
+        <h4>Cookie Policy</h4>
+        <p>
+            Like many other websites, we also use so-called “cookies”. Cookies are small text files that are transferred from a
+            website server to your hard drive. When this happens, we automatically receive certain data, such as your IP address,
+            the browser used, the operating system on your computer, and your connection to the Internet.<br>
+            Cookies cannot be used to run programs or transfer viruses to your computer. Based on the information contained in
+            the cookies, we can make navigation easier for you and make sure that our website displays correctly.<br>
+            The information we collect will never be forwarded to third parties or linked with your personal data without
+            your explicit consent.<br>
+            Of course, you can also view our website without cookies. Internet browsers are usually set up to accept cookies.
+            However, you do have the option of setting your browser so that<br>
+            1.cookies are displayed before they are saved,<br>
+            2.only certain cookies are accepted or rejected,<br>
+            3.cookies are generally rejected.<br><br>
+            This site uses different types of cookies. Some cookies are placed by third parties that appear on our pages.
+            Below you will find a list of all cookies used and you can adjust your consent. Your consent applies to the
+            following domains: www.maibornwolff.de
+        </p>
```

Comment:
because this information is not correct I would replace it with a nonsense text like lorem ipsum: https://www.loremipsum.de/